lib/yast2_vnc_shortcuts.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Shortcuts to configure remote administration with yast2 vnc
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+package yast2_vnc_shortcuts;
```

Comment:
what's the purpose of this lib when it's only used within one test module?

Reply:
I guess there was some misunderstanding of my proposal to move all shortcuts to separate file, so in case of UI changes we need to just touch that one and not anything else in the test. Similarly to `%cmd` hash, but more granular and better structured. With current proposal I also see it as a bit of over-engineering.

Reply:
I had the same question :)

Reply:
Question now is if you still have it. I'm kind of fine with current approach if and only if we continue moving shortcuts mapping to the single place, and I don't like proposed structure for it, seems to be too complex for this use-case to my taste.

Reply:
I did not merge the PR because of this unresolved discussion, @rwx788 your call now :)

Reply:
To be honest, we've been discussing topic for the test architecture with @OleksandrOrlov and this would be one of the design ideas. However, I have a high hopes on the new framework which will help us to get rid of strong dependence on shortkeys.